templates/core/form/fieldset.html.twig

```diff
@@ -72,10 +72,7 @@
     {% if has_columns %}
     <div class="form-item">
       {% if prefix %}
-        <div class="field-message">
-          {{ prefix }}
-          <div class="accolade"></div>
-        </div>
+        <span class="field-prefix">{{ prefix }}</span>
```

Comment:
volgens stijlgids is dit niet de correcte markup voor een description? https://stijlgids.stad.gent/v5/components/detail/checkboxes.html -> <span class="field-prefix"></span>

Reply:
De description zat dubbel in `{{ information.content }}` en in `{{ prefix }}`. Nu zit de description enkel nog in `{{ information.content }}`.